### PR TITLE
Dev 1750 building floor

### PR DESCRIPTION
**Pull request by yjytt59**
Building Floor Agent aims to integrate number of floors to the 3D building data.
It developed based on previous GFA agent.

### CODE REVIEW

**Comment by sm453** [comment]
Looks like #1131 will need to be merged before this PR.

**Comment by sm453** [comment]
Given the situation with #1131, the best course of action seems to be to revert all changes relating to the GFA agent and to `dev-singapore-stack` on the present branch.

**Comment by yjytt59** [comment]
#1131 has been separated in to two agents now, one is the Building Floor Agent, which is only focused on integrated number of floors to the 3D building. The other is new GFA agent, which aims to calculate the GFA and construction cost. I will pull a new request of new GFA agent soon.

File: Agents/BuildingFloorAgent/src/main/resources/buildingfloor.odba [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+xsd:    http://www.w3.org/2001/XMLSchema#
+bot:    https://w3id.org/bot#
+env: 	https://www.theworldavatar.com/kg/ontobuiltenv/
+twa:    https://www.theworldavatar.com/kg/
+
+[MappingDeclaration] @collection [[
+
+mappingId	building_floor
+target		twa:Building/{strval} env:hasNumberOfFloors twa:NumberOfFloors/{strval} .
+            twa:NumberOfFloors/{strval} a env:FloorCat{storeys_above_ground_cat} ;
```
Comment (kok-foong) [inline Agents/BuildingFloorAgent/src/main/resources/buildingfloor.odba:11]: I find defining the rdf type like this is very risky and prone to errors as the values of storeys_above_ground_cat relies heavily on the success of some non-trivial queries/logic executed by the agent.

File: Agents/BuildingFloorAgent/src/main/resources/HDBPropertyInformation.csv
Comment (sm453) [inline Agents/BuildingFloorAgent/src/main/resources/HDBPropertyInformation.csv:1]: Version control is for code, not for data! Please remove this file (and promise never to commit anything like that ever again).

Instead, please upload the file to an appropriate subfolder of `dropbox\IRP3 CAPRICORN shared folder\_TWA_Shared_Data`, and add this path to your main readme file, together with an explanation of what this is and why it is needed.

**Review by sm453 — CHANGES_REQUESTED** [review]

**Review by sm453 — APPROVED** [review]
Any change outside the `BuildingFloorAgent/` folder has been removed - this is good.

Remaining issues as discussed to be addressed by @kok-foong on a separate branch.

**Pull request merged**